examples/spice/default.spice

```diff
@@ -0,0 +1,8 @@
+f<int> fibo(int n) {
```

Comment:
Maybe it's best to keep the example consistent to a square function like the other examples?

Reply:
Done.